lunavl/sdk/errors/exceptions.py

```diff
@@ -4,7 +4,9 @@
 from functools import wraps
 from typing import Optional, Any, Callable
 
-from lunavl.sdk.errors.errors import ErrorInfo
+from FaceEngine import FSDKErrorResult  # pylint: disable=E0611,E0401
```

Comment:
E0401 does not exist
http://pylint-messages.wikidot.com/all-codes

Reply:
http://pylint.pycqa.org/en/latest/technical_reference/features.html?highlight=E0401#imports-checker-messages